client/app.jsx

```diff
@@ -43,7 +40,11 @@ class App extends React.Component {
       .catch(err => {
         console.log("GET for data failed: ", err);
       })
+  }
 
+  componentDidMount() {
+    //makes call to get data for listing
+    this.getData();
```

Comment:
I really like that you abstracted your data call from componentDidMount. Makes your componentDidMount look super clean. Maybe I should do this, hmmmmmm